### ISSUE TITLE
Behavior score bug

### ISSUE THREAD

**Issue by jeaz30** [ValveSoftware/Dota2-Gameplay#6477]
### Description

My behavior score was 9500.Played a few matches noticed i got mostly muted and wierd players.Then my conduct summary came and it showed my behavior score is 1000.I got 0 abandons and less than 3 reports.I play on only one account and never used any cheats. What to do?
My steam account:  shadow2022200125

### Example Match ID (and possibly Timestamp)

No response

### Screenshots

![2023-01-08_15-50-08](https://user-images.githubusercontent.com/48308524/211194557-f95243e6-c71e-4ead-a471-0fb7d196f563.png)

**Comment by shaileshgarg** [comment]
#6471

**Comment by shaileshgarg** [comment]
You have been reported every game, sometimes by 3-4 players.

**Comment by jeaz30** [comment]
> You have been reported every game, sometimes by 3-4 players.


I had 9500 points. How did I get up to 1000 points in 15 games? Even considering these reports.

**Comment by jeaz30** [comment]
Can anyone help resolve this issue? Where is the Dota 2 support? I was undeservedly given this decency. Please solve this problem

**Comment by GauravDe-git** [comment]
+1 , My Friend got same issue

**Comment by jeaz30** [comment]
@jeffhill can you help with this question , because more people got it wrong because of a bug

**Comment by usser1d** [comment]
Im the guy from #6476 

All i know i used only one account, never used cheats, and my behavior ingame is decent.I got a link here where i can see my reports from the summarys and i see very few reports.So yes, i will call it a BUG.
Im waiting for someone qualified to speak about this topic.

If you got some biased opinions you wish to share, best you take it to redit and not here.(since the bug didnt happend to you, and you are not qualified)
Thanks

**Comment by sievenn** [comment]
im from #6488

All I can say is that I never liked playing on second accounts, I always played on the main one. Before writing my issue, I analyzed the previous ones, where I saw that I was not the only one who faced this problem. Above, people wrote about the same problem, and the description is similar to mine.

**Comment by jeaz30** [comment]
> This is not a bug, this is the system working as intended to catch your attempts to dodge your behavior score. People need to stop falsely reporting it as a bug when it's obvious they're using alts (in dota2 and here on github) in case there actually were false positives that need to get found

Why are you judging me for something I didn't do. Check the IP address, I play on the same account. Why do you write that these are second accounts? If it's a bug, then it needs to be fixed. I'm not alone with this.

**Comment by sievenn** [comment]
> No, he's been reported 1 or 3 games out of every 15. Conduct summary is last 15 games, not per game. There's even multiple columns that show this.
> 
> OP was caught by low behavior score alt detection, but your analysis of the situation is just wrong

No, he's been reported 1 or 3 games out of every 15. Conduct summary is last 15 games, not per game. There's even multiple columns that show this.

OP was caught by low behavior score alt detection, but your analysis of the situation is just wrong.
#6488

**Comment by sievenn** [comment]
> No, he's been reported 1 or 3 games out of every 15. Conduct summary is last 15 games, not per game. There's even multiple columns that show this.
> 
> OP was caught by low behavior score alt detection, but your analysis of the situation is just wrong.

If I understand everything correctly, then this is just a mistake? Then can you answer me? I created an issue 2 days ago.

#6488 

@MemoriesOfManaLeak
